hands-on-exercise/3-cosmjs-adv/3-external-gui.md

```diff
@@ -4,8 +4,6 @@ order: 4
 description: Find a checkers GUI before integrating with CosmJS
 tags: 
   - guided-coding
-  - cosmos-sdk
```

Comment:
@xavierlepretre if possible, I would recommend adding always one tag that relates to the type (guided coding, tutorials, concept) and content area (IBC, CosmJS, etc.). If only guided coding fits, I would leave it like that and merge the PR. :)

Reply:
The thing is that on this very page, we are saying "let's pick something elsewhere and make sure it will work". That's React.
Additional benefit is that, on the left menu, it will look different, and so break the monotony. At the moment, they all look the same and therefore the visual meaning of tags is lost.

Reply:
ok, I would say leave the change in. 😉